libs/storage/Tsavorite/cs/src/core/ClientSession/LockableContext.cs

```diff
@@ -53,30 +53,22 @@ internal LockableContext(ClientSession<TKey, TValue, TInput, TOutput, TContext,
         public int CompareKeyHashes<TLockableKey>(ref TLockableKey key1, ref TLockableKey key2) where TLockableKey : ILockableKey => clientSession.CompareKeyHashes(ref key1, ref key2);
 
         /// <inheritdoc/>
-        public void SortKeyHashes<TLockableKey>(TLockableKey[] keys) where TLockableKey : ILockableKey => clientSession.SortKeyHashes(keys);
-
-        /// <inheritdoc/>
-        public void SortKeyHashes<TLockableKey>(TLockableKey[] keys, int start, int count) where TLockableKey : ILockableKey => clientSession.SortKeyHashes(keys, start, count);
+        public void SortKeyHashes<TLockableKey>(Span<TLockableKey> keys) where TLockableKey : ILockableKey => clientSession.SortKeyHashes(keys);
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         internal static bool DoManualLock<TSessionFunctionsWrapper, TLockableKey>(TSessionFunctionsWrapper sessionFunctions, ClientSession<TKey, TValue, TInput, TOutput, TContext, TFunctions, TStoreFunctions, TAllocator> clientSession,
-                                                                   TLockableKey[] keys, int start, int count)
+                                                                   ReadOnlySpan<TLockableKey> keys)
             where TSessionFunctionsWrapper : ISessionFunctionsWrapper<TKey, TValue, TInput, TOutput, TContext, TStoreFunctions, TAllocator>
             where TLockableKey : ILockableKey
         {
-            // The key codes are sorted, but there may be duplicates; the sorting is such that exclusive locks come first for each key code,
-            // which of course allows the session to do shared operations as well, so we take the first occurrence of each key code.
-            // This is the same as DoManualTryLock but without timeout; it will keep trying until it acquires all locks or the hardcoded retry limit is reached.
```

Comment:
Why were these comments (here and DoManualTryLock) removed? The information is useful